.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -69,11 +62,7 @@ Please describe the tests that you ran to verify your changes.
 ## Checklist:
 
 - [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
-- [ ] I have commented on my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
-- [ ] My changes generate no new warnings
-- [ ] Any dependent changes have been merged and published in downstream modules
 - [ ] I have removed all unnecessary comments and debug log statements
```

Comment:
```suggestion
```

Reply:
makes sense to remove this, looks good with the changes - lightens the PR template.